fml/platform/fuchsia/message_loop_fuchsia.cc

```diff
@@ -0,0 +1,38 @@
+// Copyright 2019 The Flutter Authors. All rights reserved.
```

Comment:
Please copy and paste the existing copyright header - it needs to be "Copyright 2013".

Reply:
Ahh OK, I did not know that